docs/tutorial/part-one/index.md

```diff
@@ -200,17 +202,17 @@ export default () => (
 
 ![Adding Header component](06-header-component.png)
 
-In the browser, the “About Gatsby” header text should now be replaced with “This is a header.” But you don’t want the “About” page to say “This is a header.” You want it to say, “About Gatsby”.
+ब्राउज़र में, “About Gatsby” हेडर टेक्स्ट अब “This is  header.” के साथ रीप्लेस हो जाना चाहिए। लेकिन आप यह नहीं चाहते हैं कि “About" पेज यह कहे कि "This is a header.” आप इसे कहना चाहते हैं, “About Gatsby”।
```

Comment:
`“This is  header.”` => `“This is a header.”`

Reply:
You missed this

Reply:
This is missed again.

Reply:
done